### PR TITLE
#4 Set up Storybook + React USWDS

### LINKED ISSUE #4: Set up Storybook + ReactUSWDS
In order to prepare for prototype development, install the following:
- Storybook (React) https://storybook.js.org/
- ReactUSWDS

Additional info should be added to the README about how to start up & access Storybook

### PR DESCRIPTION
**Pull request by abbyoung**
## Description 

* Sets up Storybook instance that is compatible with React USWDS, css modules, and a few basic add-ons.
* Adds React USWDS to Storybook and NextJS so we can pull in components
* Creates components directory and Logo sample component to establish pattern
* Updates README for how to run Storybook

Fixes #4 

## Review Notes
* Do any directories need to be moved around? e.g. components
* Any additional setup you'd like to see?

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import '@trussworks/react-uswds/lib/uswds.css'
+import '@trussworks/react-uswds/lib/index.css';
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
```
Comment (suzubara) [inline .storybook/preview.js:10]: these will automatically show color/date picker controls for args (props) that match the regex, right? just making sure I am reading this right 😄

File: package.json [hunk 2]
```diff
@@ -5,15 +5,28 @@
...
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "storybook": "start-storybook -p 6006 -s ./public",
+    "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@storybook/addon-a11y": "^6.3.0",
+    "@trussworks/react-uswds": "^2.0.0",
+    "@zeit/next-css": "^1.0.1",
```
Comment (suzubara) [inline package.json:15]: what's this dependency for? googling is telling me that this package has been deprecated

**Review by suzubara — APPROVED** [review]
🎉  looks good, I think this checks all the boxes we wanted for this issue!

File: package.json (hunk 2 above)
Comment (suzubara) [reply package.json:15]: if we can delete it then we should

Comment (abbyoung) [reply package.json:15]: good catch, thank you! this was from a debugging session and is not needed, you're right

File: .storybook/preview.js (hunk 1 above)
Comment (abbyoung) [reply .storybook/preview.js:10]: this was a storybook default. looking at the docs, you are correct! https://storybook.js.org/docs/riot/essentials/controls#custom-control-type-matchers

**Pull request merged**